exporter/exporter_test.go

```diff
@@ -34,9 +34,37 @@ var _ = Describe("CheckForNewApps", func() {
 		Eventually(fakeWatcherManager.AddWatcherCallCount).Should(Equal(1))
 	})
 
+	It("does not create a new appWatcher if the app state is stopped", func() {
+		fakeClient.ListAppsByQueryReturns([]cfclient.App{
+			{Guid: "33333333-3333-3333-3333-333333333333", Instances: 1, Name: "foo", SpaceURL: "/v2/spaces/123", State: "STOPPED"},
+		}, nil)
+
+		e := exporter.New(fakeClient, fakeWatcherManager)
+
+		go e.Start(100 * time.Millisecond)
+
+		Eventually(fakeWatcherManager.AddWatcherCallCount).Should(Equal(0))
```

Comment:
```suggestion
		Consistently(fakeWatcherManager.AddWatcherCallCount).Should(Equal(0))
```
`Eventually` passes as soon as it detects a 0, but we want to assert that it never proceeds to 1, so `Consistently` feels like the right choice here